README.md

```diff
@@ -56,11 +118,77 @@ repository which contains following:
+in a terminal with Python 3.5+ installed, run:
 
 ```bash
 pip install cookiecutter
 cookiecutter gh:CouncilDataProject/cookiecutter-cdp-deployment
 ```
 
+_Note: This will only create the basic repository. You will still need to setup
+Google Cloud and Pulumi accounts._
+
+### Google Cloud
+
+All of your deployments data and some data processing will be done using
+Google Cloud Platform (GCP).
+
+-   Your deployment's provided and generated data (meeting dates,
+    committee names, councilmember details, etc) will live in
+    [Firestore](https://cloud.google.com/firestore).
+-   Your deployment's generated files (audio clips, transcripts, etc.)
+    will live in [Filestore](https://cloud.google.com/filestore).
+-   When provided a video without closed captions, the audio from the provided video
+    will be processed using [Speech-to-Text](https://cloud.google.com/speech-to-text).
+
+All of these resources will be set up for you using [Pulumi](#pulumi) but you will
+need to make
+
+## Cost
+
+CDP was created and maintained by a group of people working on it in their free time.
+We didn't want to pay extreme amounts of money so why should you?
+
+So to that end, we try to make CDP as low cost as possible.
+Many of the current features are entirely free as long as the repo is open source:
+
+-   Event Processing (GitHub Actions)
+-   Event and Legislation Indexing (GitHub Actions)
+-   Web Hosting (GitHub Pages)
+-   Infrastructure State Management (Pulumi)
+
+The backend resources and processing are the only real costs and depend on usage.
+The more users that use your web application, the more the database and file storage
+costs. The CDP-Seattle monthly averages below are the averages for the most utilized
+months of it's existance so take these as close to upper-bounds.
+
+-   [Cloud Firestore Pricing](https://firebase.google.com/pricing/)
+    _CDP-Seattle monthly average: ~$8.00_
+-   [Google Storage Pricing](https://cloud.google.com/storage/pricing#price-tables)
+    _CDP-Seattle monthly average: ~$3.00_
+-   [Google Speech-to-Text Pricing](https://cloud.google.com/speech-to-text/pricing)
+    _CDP-Seattle monthly average: ~$22.00_
+
+**Total Average Monthly Cost**: $33.00
+
+### Speech-to-Text
+
+You may not need to use speech-to-text! In the case your municipalicity provides closed
+caption files in a format we support parsing and cleaning, we can use those files
+instead of using speech-to-text. When using closed caption files for transcription
+generation. CDP-Seattle speech-to-text costs dropped to ~$2.00 / month because an
+occasional meeting didn't have closed captions. You can attach a
+[`closed_caption_uri`](https://councildataproject.github.io/cdp-backend/cdp_backend.pipeline.html#cdp_backend.pipeline.ingestion_models.Session)
+to the `Session` object during event ingestion.
+
+With speech-to-text cost removed, total average monthly cost for CDP-Seattle is ~$11.00.
+
+### Future Processing Features
```

Comment:
It might be worth noting how updates happen - do they happen in `cdp-backend` and therefore are automatically applied to my deployment? Do I get notified somehow?

Reply:
The reason for the having the issue template is that I can see it feasible to automate the PR to add the deployment details.

if you have a template with the repo name, list of maintainers, supported municipality, and some other metadata, you can create a bot to automatically add those details to the list and create a PR.

(Overkill probably but it would be neat)

Reply:
Also we do have a slightly built up system for pulling in upgrades already. In the generated repos, Makefile there is a command called `update-from-cookiecutter` that reruns the cookiecutter in place. It works pretty well but you have to be good about git because after it runs you as a maintainer need to cherrypick which changes to keep and to discard.

It has worked so far in maintaining the CouncilDataProject/example repo.